src/app/index.js

```diff
@@ -7,6 +7,11 @@ angular.module('uvpBuilderWeb', ['ngAnimate', 'ngCookies', 'ngTouch', 'ngSanitiz
         url: '/',
         templateUrl: 'app/main/main.html',
```

Comment:
this is always considere to be the `home` page so, we should change it to `app/home/home.html`, and the controller has to be `HomeController`